.vscode/launch.json

```diff
@@ -32,7 +32,7 @@
     },
     {
       "name": "Launch Web Extension",
-      "type": "pwa-extensionHost",
+      "type": "extensionHost",
```

Comment:
Apparently `pwa-extensionHost` is deprecated even though the documentation still refers to it? Not sure what's up there